manifests/machineconfigdaemon/clusterrole.yaml

```diff
@@ -19,6 +19,9 @@ rules:
   resourceNames: ["privileged"]
   resources: ["securitycontextconstraints"]
   verbs: ["use"]
+- apiGroups: [""]
+  resources: ["configmaps", "secrets"]
```

Comment:
Seems like you'll want ConfigMap get/list/watch, but I don't see you adding any Secrets-touching logic in this pull.  Maybe you can drop `secrets` here?

I'm not sure if there's a way to restrict to `openshift-config-managed`.  [Docs][1] include:

> ClusterRoles have several uses. You can use a ClusterRole to:
>
> 1. define permissions on namespaced resources and be granted access within individual namespace(s)
> 2. ...

But I don't see them explaining how to achieve that "within individual namespace(s)" scoping.

[1]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole

Reply:
hmmm good point, I can look into namespacing this. Secrets may be necessary for testing? probably not though